netci.groovy

```diff
@@ -1251,6 +1251,11 @@ combinedScenarios.each { scenario ->
                     // The tuples (LinuxARMEmulator, other architectures) are not handled and control returns
                     def isLinuxEmulatorBuild = false
                     if (os == 'LinuxARMEmulator' && architecture == 'arm') {
+                        // Cross Builds only in Debug and Release modes allowed
+                        if ( configuration == 'Checked' ) {
```

Comment:
`lowerConfiguration` is defined later in line `1420`. Before that definition, `configuration` is the variable used (as in line `1293`).